Mathlib/Analysis/SpecificLimits/Normed.lean

```diff
@@ -280,71 +285,83 @@ section Geometric
   rw [hasSum_iff_tendsto_nat_of_summable_norm]
   · simpa [geom_sum_eq, xi_ne_one, neg_inv, div_eq_mul_inv] using A
-  · simp [norm_pow, summable_geometric_of_lt_1 (norm_nonneg _) h]
-#align has_sum_geometric_of_norm_lt_1 hasSum_geometric_of_norm_lt_1
-
-theorem summable_geometric_of_norm_lt_1 (h : ‖ξ‖ < 1) : Summable fun n : ℕ ↦ ξ ^ n :=
-  ⟨_, hasSum_geometric_of_norm_lt_1 h⟩
-#align summable_geometric_of_norm_lt_1 summable_geometric_of_norm_lt_1
-
-theorem tsum_geometric_of_norm_lt_1 (h : ‖ξ‖ < 1) : ∑' n : ℕ, ξ ^ n = (1 - ξ)⁻¹ :=
-  (hasSum_geometric_of_norm_lt_1 h).tsum_eq
-#align tsum_geometric_of_norm_lt_1 tsum_geometric_of_norm_lt_1
-
-theorem hasSum_geometric_of_abs_lt_1 {r : ℝ} (h : |r| < 1) : HasSum (fun n : ℕ ↦ r ^ n) (1 - r)⁻¹ :=
-  hasSum_geometric_of_norm_lt_1 h
-#align has_sum_geometric_of_abs_lt_1 hasSum_geometric_of_abs_lt_1
-
-theorem summable_geometric_of_abs_lt_1 {r : ℝ} (h : |r| < 1) : Summable fun n : ℕ ↦ r ^ n :=
-  summable_geometric_of_norm_lt_1 h
-#align summable_geometric_of_abs_lt_1 summable_geometric_of_abs_lt_1
-
-theorem tsum_geometric_of_abs_lt_1 {r : ℝ} (h : |r| < 1) : ∑' n : ℕ, r ^ n = (1 - r)⁻¹ :=
-  tsum_geometric_of_norm_lt_1 h
-#align tsum_geometric_of_abs_lt_1 tsum_geometric_of_abs_lt_1
+  · simp [norm_pow, summable_geometric_of_lt_one (norm_nonneg _) h]
+#align has_sum_geometric_of_norm_lt_1 hasSum_geometric_of_norm_lt_one
+@[deprecated] alias hasSum_geometric_of_norm_lt_1 := hasSum_geometric_of_norm_lt_one
+
+theorem summable_geometric_of_norm_lt_one (h : ‖ξ‖ < 1) : Summable fun n : ℕ ↦ ξ ^ n :=
+  ⟨_, hasSum_geometric_of_norm_lt_one h⟩
+#align summable_geometric_of_norm_lt_1 summable_geometric_of_norm_lt_one
+@[deprecated] alias summable_geometric_of_norm_lt_1 := summable_geometric_of_norm_lt_one
+
+theorem tsum_geometric_of_norm_lt_one (h : ‖ξ‖ < 1) : ∑' n : ℕ, ξ ^ n = (1 - ξ)⁻¹ :=
+  (hasSum_geometric_of_norm_lt_one h).tsum_eq
+#align tsum_geometric_of_norm_lt_1 tsum_geometric_of_norm_lt_one
+@[deprecated] alias tsum_geometric_of_norm_lt_1 := tsum_geometric_of_norm_lt_one
+
+theorem hasSum_geometric_of_abs_lt_one {r : ℝ} (h : |r| < 1) :
+    HasSum (fun n : ℕ ↦ r ^ n) (1 - r)⁻¹ :=
+  hasSum_geometric_of_norm_lt_one h
+#align has_sum_geometric_of_abs_lt_1 hasSum_geometric_of_abs_lt_one
+@[deprecated] alias hasSum_geometric_of_abs_lt_1 := hasSum_geometric_of_abs_lt_one
+
+theorem summable_geometric_of_abs_lt_one {r : ℝ} (h : |r| < 1) : Summable fun n : ℕ ↦ r ^ n :=
+  summable_geometric_of_norm_lt_one h
+#align summable_geometric_of_abs_lt_1 summable_geometric_of_abs_lt_one
+@[deprecated] alias summable_geometric_of_abs_lt_1 := summable_geometric_of_abs_lt_one
+
+theorem tsum_geometric_of_abs_lt_one {r : ℝ} (h : |r| < 1) : ∑' n : ℕ, r ^ n = (1 - r)⁻¹ :=
+  tsum_geometric_of_norm_lt_one h
+#align tsum_geometric_of_abs_lt_1 tsum_geometric_of_abs_lt_one
+@[deprecated] alias tsum_geometric_of_abs_lt_1 := tsum_geometric_of_abs_lt_one
```

Comment:
Yikes, github making our life harder with this diff. @alexjbest can you use Leaff to confirm that this PR only does what it claims to do? Can you even check the `@[deprecated]` stuff does what it's supposed to do?

Reply:
It looks fine to me from a visual inspection.

Reply:
Frankly you could leave out the deprecated lines but it's probably kinder to keep them in I guess.)

Reply:
I imagine these results are likely to be used in downstream projects, and there it is nice to be told automatically after updating Mathlib that the names are now deprecated, with the offer of fixing them (rather than getting an "unknown identifier" error and having to consult the change log).

Reply:
As I say, it is kinder - I like that we have the option and I like that you've done it.